src/screens/Auth/CreatePasswordScreen/CreatePasswordScreen.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react'
+import {
+  Text,
+  View,
+  TouchableOpacity,
+} from 'react-native'
+import styles from '../../Auth/CreatePasswordScreen/styles'
+import BlackHeading from '../../../shared_components/BlackHeading/BlackHeading'
+import GreyTextInput from '../../../shared_components/GreyTextInput/GreyTextInput'
+import PinkButton from '../../../shared_components/PinkButton/PinkButton'
+import { NavigationScreenProp, NavigationState } from 'react-navigation'
+
+interface Props {
+  navigation: NavigationScreenProp<NavigationState>
+}
+
+export default function CreatePasswordScreen(props: Props) {
+  const [password, setPassword] = useState('')
+  const [confirmPassword, setConfirmPassword] = useState('')
+  const {navigate} = props.navigation 
+
+  function changePassword(password: string){
+    setPassword(password)
+  }
+
+  function changeConfirmPassword(confirmPassword: string){
+    setConfirmPassword(confirmPassword)
+  }
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.main}>
+        <BlackHeading title="Create a Password" />
+        <Text style={styles.regularText}>Keep this secure!</Text>
+        <GreyTextInput changeTextContent={changePassword} placeholder="Password (8+ characters)" inputType='password'/>
+        <GreyTextInput changeTextContent={changeConfirmPassword} placeholder="Confirm Password" inputType='password'/>
```

Comment:
I seem to have forgotten to save this comment which was referenced many times below, but changePassword() and changeConfirmPassword() can removed and done inline lie this.
```js
<GreyTextInput changeTextContent={(text) => {
        setPassword(text)
    }
} placeholder="Password (8+ characters)" inputType='password'/>

```

Reply:
Sorry, it's my bad. I saw these comments but didn't fix them all at a time yesterday. I will react with thumbs up and leave a comment to let you know when I finish solving all the problems ;)

Reply:
you're fine! I thought this was my fault bc I didn't see it in my review. not a big deal don't worry. You don't need to comment unless you have questions, thumbs up is good!